tor/strings/responses.py

```diff
@@ -52,3 +54,19 @@
     'process your claim as normal.'
     '\n\n---\n\n{}'
 )
+
+thumbs_up_gifs = [
+    'https://media.giphy.com/media/oBPOP48aQpIxq/giphy.gif',
+    'https://media.giphy.com/media/vMnuZGHJfFSTe/giphy.gif',
+    'https://media.giphy.com/media/XreQmk7ETCak0/giphy.gif',
+    'https://media.giphy.com/media/ClcWrARkrq1GM/giphy.gif',
+    'http://i.imgur.com/QMPRHW9.gif',
+    'https://media.giphy.com/media/j5QcmXoFWl4Q0/giphy.gif',
+    'http://www.reactiongifs.com/r/a-b-yay.gif',
+    'https://media.tenor.com/images/7dcd26f4d9a2eaa246cd4ea9967d2336/tenor.gif',
+    'http://media0.giphy.com/media/wTzPSxZBqSove/giphy.gif'
+]
+
+youre_welcome = (  # just format that sucker as a link and send it back
+    '[Hey, you\'re welcome!]({})'.format(random.choice(thumbs_up_gifs))
```

Comment:
Not catastrophical, but this would only evaluate once and use the same gif every instance of the bot that is run.

Reply:
I distinctly remember thinking "I wonder if that wouldn't work" and then dismissing it. Great catch!

Reply:
Yep, it would definitely insert one image consistently until the bot is completely restarted. Possibly python's caching mechanisms (`*.pyc`) might mess with this too, not sure.